utils/src/com/goide/GoEnvironmentUtil.java

```diff
@@ -13,15 +12,22 @@
 
 public class GoEnvironmentUtil {
   public static final String GO_EXECUTABLE_NAME = "go";
-  
+  public static final String GAE_EXECUTABLE_NAME = "goapp";
+
   private GoEnvironmentUtil() {
   }
 
-  @NotNull
+  @Nullable
```

Comment:
I guess I can change it back to not returning null and if the ```canExecute()``` call is successful then return the path with ```getAbsolutePath()``` already called.

However, I'd prefer for the clients to have a way to check and signal if something is missing from their SDK directory even after the SDK was configured the first time around as we rely on things such as ```go fmt``` or ```go run``` and I think it's not nice not having any feedback for the user in case something went wrong as to why it happened.